browser/brave_origin/brave_origin_service_factory.cc

```diff
@@ -193,6 +194,13 @@ BraveOriginServiceFactory::BuildServiceInstanceForBrowserContext(
                          g_browser_process->local_state());
   }
 
+  // Lazy initialization of AdBlockOnlyModePolicyManager
+  auto* ad_block_only_mode_policy_manager =
```

Comment:
why is adblock only mode inside brave origin?

Reply:
I did it after discussion with @bbondy that BraveOriginPolicyManager will be generalized to BravePolicyManager. And AdBlockOnlyModePolicyManager can stay here until it is done. There is a ticket for generalization: https://github.com/brave/brave-browser/issues/50077